lib/api/http_util.dart

```diff
@@ -26,3 +73,8 @@ class HttpUtil {
     return code >= 200 && code < 300;
   }
 }
+
+enum SERVER {
```

Comment:
should this enum put in module package?

Reply:
I think put here, SERVER current only used in httpUtil